pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[tool.ruff]
+# In addition to the standard set of exclusions, omit all tests, plus a specific file.
+extend-exclude = [
+    "*/migrations/*",
+    "*/templates/*",
+    "*/static/*",
+    "*.html",
+    "*.css",
+    "*.js",
+    "__init__.py",
+]
+force-exclude = true
```

Comment:
What does this do?

Reply:
im gonna quote the docs to be clear:

Whether to enforce exclude and extend-exclude patterns, even for paths that are passed to Ruff explicitly. Typically, Ruff will lint any paths passed in directly, even if they would typically be excluded. Setting force-exclude = true will cause Ruff to respect these exclusions unequivocally.

This is useful for pre-commit, which explicitly passes all changed files to the ruff-pre-commit plugin, regardless of whether they're marked as excluded by Ruff's own settings.

Reply:
ok cool, thanks